src/info.plist

```diff
@@ -3,9 +3,9 @@
 <plist version="1.0">
 <dict>
 	<key>bundleid</key>
-	<string>com.AmosManneschmidt.callorsmscontact</string>
```

Comment:
Ordinarily I would suggest not changing the BundleID as I think it may be used with Alfred-Workflow during auto updates to not have more than one copy of the same workflow installed, however, on looking back I do not believe I ever properly configured autoupdates anyway so I don't think it really matters because the update will have to be manual anyway.

Reply:
Feel free to change the bundle ID back if you would like, or we can make a new one. I just changed it because I wasn't sure if I was going to be the sole maintainer or not from here on out.

Reply:
I'll leave it for now. My most up-to-date Mac is a mid 2010 MacBook Pro and I'm still actively using an iPhone 6. My phone is in queue to be updated next year when the next model comes out or whenever I break the 6; whichever is sooner. My most recent laptop purchase a year ago wasn't a mac because Apple literally didn't (and still doesn't) make a non-mobile-processor-quality VR capable laptop without using an eGPU (which is an untenable solution to me). However, I do expect Apple to release VR/AR capable laptops in the future if they ever release their AR headset solution as not being able to develop for your own ecosystem within your own ecosystem would be hugely embarrassing. At that point in time (or if I decide to Hackintosh my current laptop before then) I will likely switch back but that is still probably at least 2 years out if rumors are believed.

Reply:
It'd be nice if there wasn't a need for an eGPU, and I don't blame you for not having one. It's too expensive anyway. The new phones aren't too special, so you aren't missing much there either. 

I haven't dabbled much in the AR/VR world, but I can tell that it is picking up more steam each and every day. I hope Apple decides to invest more resources into that path due to the potential it may have.

I also changed the ID back on my fork too.